root/edit/details/EditWork.js

```diff
@@ -0,0 +1,186 @@
+  },
+};
+
+const EditWork = ({edit}: {edit: Props}) => {
+  const display = edit.display_data;
+  const comment = display.comment;
+  const iswc = display.iswc;
+  const name = display.name;
+  const type = display.type;
+  const language = display.language;
+  const languages = display.languages;
+  const attributes = display.attributes ?? {};
+  return (
+    <table className="details edit-work">
+      <tr>
+        <th>{addColonText(l('Work'))}</th>
+        <td colSpan="2">
+          <DescriptiveLink entity={display.work} />
+        </td>
+      </tr>
+      {name ? (
+        <WordDiff
+          label={addColonText(l('Name'))}
+          newText={name.new}
+          oldText={name.old}
+        />
+      ) : null}
+      {comment ? (
+        <WordDiff
+          label={addColonText(l('Disambiguation'))}
+          newText={comment.new ?? ''}
+          oldText={comment.old ?? ''}
+        />
+      ) : null}
+      {iswc ? (
+        <Diff
+          label={addColonText(l('ISWC'))}
+          newText={iswc.new ?? ''}
+          oldText={iswc.old ?? ''}
+        />
+      ) : null}
+      {type ? (
+        <FullChangeDiff
+          label={addColonText(l('Work type'))}
+          newContent={
+            type.new?.name ? lp_attributes(type.new.name, 'work_type') : ''
+          }
+          oldContent={
+            type.old?.name ? lp_attributes(type.old.name, 'work_type') : ''
+          }
+        />
+      ) : null}
+      {language ? (
+        <FullChangeDiff
+          label={addColonText(l('Language'))}
+          newContent={l_languages(language.new?.name ?? '')}
+          oldContent={l_languages(language.old?.name ?? '')}
+        />
+      ) : null}
+      {languages ? (
+        <Diff
+          label={addColonText(l('Lyrics Languages'))}
+          newText={commaOnlyListText(languages.new.map(
+            language => l_languages(language),
+          ))}
+          oldText={commaOnlyListText(languages.old.map(
+            language => l_languages(language),
+          ))}
+          split=", "
+        />
+      ) : null}
+      {edit.data.new.attributes
+        ? Object.keys(attributes).sort().map((attributeName) => (
+          <tr key={attributeName}>
+            <th>
+              {addColonText(lp_attributes(
+                attributeName,
+                'work_attribute_type',
+              ))}
+            </th>
+            <td className="old">
+              {attributes[attributeName].old.length > 0
+                ? (
+                  <ul>
+                    {attributes[attributeName].old.map((attribute, index) => (
+                      <li key={attribute}>
+                        <DiffSide
+                          filter={DELETE}
+                          newText={
+                            lp_attributes(
+                              attributes[attributeName].new[index] || '',
+                              'work_attribute_type_allowed_value',
+                            )
```

Comment:
The attribute names and values are already all translated in the display data (note the use of `l_value` here):
https://github.com/metabrainz/musicbrainz-server/blob/1555e503d2f10a404ec81b0314def70e89eff89e/lib/MusicBrainz/Server/Edit/Work/Edit.pm#L168-L169

So all the `lp_attributes` calls can be removed here.

Reply:
Is there a benefit of doing it like that? I'd much prefer we were consistent and either tried to always translate the stuff already on the Perl layer, or always on React...

Reply:
Not really, just that it's already written like that so requires less changes. Another minor problem is that the translation should only be done if a `value_id` is set, like `l_value` has, but we don't have the `value_id` here to check that.